app/src/main/java/net/squanchy/support/lang/Func0.java

```diff
@@ -0,0 +1,6 @@
+package net.squanchy.support.lang;
+
+public interface Func0<T> {
```

Comment:
Sure, I kept them because I did not know what we were going to use :-) Removing them